language/stdlib/modules/libra_system.mvir

```diff
@@ -713,7 +713,7 @@ module LibraSystem {
 
          index = 0;
 
-         while (copy(index) < copy(num_validators)) {
+         while (copy(index) < copy(num_validators) && copy(amount_to_distribute_per_validator) > 0) {
```

Comment:
Since `amount_to_distribute_per_validator` is a constant, can't we check this earlier (+ avoid checking it on every loop iteration)? I would do something like:

```
if (copy(amount_to_distribute_per_validator) < copy(num_validators) {
  return;
}
```

Reply:
Tim wisely points out this won't work because `collected_fees` will be unmoved in this case, but that we can check the same condition earlier.